cmd/kube-apiserver/app/options/options_test.go

```diff
@@ -108,7 +108,6 @@ func TestAddFlags(t *testing.T) {
 		"--etcd-certfile=/var/run/kubernetes/etcdce.crt",
 		"--etcd-cafile=/var/run/kubernetes/etcdca.crt",
 		"--http2-max-streams-per-connection=42",
-		"--kubelet-read-only-port=10255",
```

Comment:
should this file get reverted? it's testing that the flag is functional, which it still should be, right?

Reply:
we can keep this flag ut.